app/src/main/java/org/connectbot/util/VolumePreference.java

```diff
@@ -56,7 +57,13 @@ protected View onCreateDialogView() {
 		sb.setMax(100);
 		sb.setProgress((int) (getPersistedFloat(
 				PreferenceConstants.DEFAULT_BELL_VOLUME) * 100));
-		sb.setPadding(10, 10, 10, 10);
+
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+			sb.setPadding(75, 70, 75, 10);
```

Comment:
No this did not come from the material spec, was just adjusting the padding so that the Seekbar has even padding and not hugging the top edge of the dialog.

Looking again at the [material spec](https://www.google.com/design/spec/components/sliders.html#sliders-continuous-slider) though we could do better by adding a volume icon to the left of the Seekbar. I could add this in another commit and add to the PR if you would like that change :)